tests/migration_test.go

```diff
@@ -1138,6 +1173,86 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 
 			})
 		})
+
+		Context("without CPU properly set", func() {
+
+			table.DescribeTable("should block migration with feature-gate CPUMigrationGate enabled", func(cpuModel string) {
```

Comment:
Good use for table here!

nit: I think this one and the following table could be collapsed into one with an additional `boolean` controlling whether to enable or to disable the feature gate. This would further reduce duplicate code I think.

Reply:
I agree! I have reduced the code duplicity. I like the two tables more because it is more verbose what it is tested there. Therefore I reduced the duplicity by creating sub-functions.